tools/scripts/checkSurrogateExists.py

```diff
@@ -0,0 +1,36 @@
+#!/bin/python3
+import os
+import sys
+
+srcdir = "../../src/catch2"
```

Comment:
There is already a centralized way to get the path to the top level folder in, IIRC, `releaseCommon` or `scriptCommon.py`

Reply:
I will refactor to use one of the two mentioned scripts.